spec/services/linked_data/mapper/term_results_mapper_service_spec.rb

```diff
@@ -2,56 +2,93 @@
 
 RSpec.describe Qa::LinkedData::Mapper::TermResultsMapperService do
   let(:graph) { Qa::LinkedData::GraphService.load_graph(url: 'http://local.data') }
-  let(:predicate_map) do
+  let(:subjects) { subject.map { |result| result[:uri].first.to_s } }
+
+  let(:expected) do
     {
-      uri: :subject_uri,
-      id: RDF::URI.new('http://purl.org/dc/terms/identifier'),
-      label: RDF::URI.new('http://www.w3.org/2004/02/skos/core#prefLabel'),
-      altlabel: RDF::URI.new('http://www.w3.org/2004/02/skos/core#altLabel'),
-      narrower: RDF::URI.new('http://www.w3.org/2004/02/skos/core#narrower'),
-      broader: RDF::URI.new('http://www.w3.org/2004/02/skos/core#broader'),
-      sameas: RDF::URI.new('http://www.w3.org/2004/02/skos/core#exactMatch')
+      uri: ['http://aims.fao.org/aos/agrovoc/c_9513'],
+      id: ['http://aims.fao.org/aos/agrovoc/c_9513'],
+      label: ['buttermilk'],
+      altlabel: ['yummy'],
+      broader: ['http://aims.fao.org/aos/agrovoc/c_4830'],
+      narrower: [],
+      sameas: ['http://cat.aii.caas.cn/concept/c_26308',
+               'http://lod.nal.usda.gov/nalt/20627',
+               'http://d-nb.info/gnd/4147072-2']
     }
   end
-  let(:subjects) { subject.map { |result| result[:uri].first.to_s } }
+
+  let(:results) do
+    results = {}
+    subject.each { |k, v| results[k] = v.map(&:to_s) }
```

Comment:
Rewrite RDF::Literals as strings because two instances of RDF::Literal with the same string value are not pass the `==` equality test, but their `.to_s` string values do pass equality tests.